build/build_scripts/import_test_dataset1.sh

```diff
@@ -8,6 +8,8 @@ spt workflow configure --local --workflow='reduction visual' --study-name='Melan
 nextflow run .
 rm -f .nextflow.log*; rm -rf .nextflow/; rm -f configure.sh; rm -f run.sh; rm -f main.nf; rm -f nextflow.config; rm -rf work/; rm -rf results/
 
+spt cggnn upload --spt_db_config_location build/db/.spt_db.config.local --importances_csv_path test/test_data/adi_preprocessed_tables/dataset1/importances.csv
```

Comment:
The path `test/test_data/adi_preprocessed_tables/...` is meant to point to input datasets, not intermediate computation data (even if fake). These `importances.csv` files should probably go in `build/`.